modules/shortcodes/tweet.php

```diff
@@ -111,6 +111,12 @@ static public function jetpack_tweet_url_extra_args( $provider, $url, $args = ar
 		// Twitter doesn't support maxheight so don't send it
 		$provider = remove_query_arg( 'maxheight', $provider );
 
+		// Add Twitter partner ID to track embeds from Jetpack
+		$partner = apply_filters( 'jetpack_twitter_partner_id', 'jetpack' );
+		if ( false !== $partner ) {
```

Comment:
This may be better to do as `if ( ! empty( $partner ) ) {` -- some folks may filter and return an empty string or null instead.